types/src/front/data_source.ts

```diff
@@ -121,37 +124,3 @@ export function isDataSourceNameValid(name: string): Result<void, string> {
 
   return new Ok(undefined);
 }
-
-export function isConnectorProviderAllowed(
```

Comment:
This was a duplicate of the `isConnectorProviderAllowedForPlan` method, I removed it.